ui/src/app/machines/views/MachineList/FilterAccordion/FilterAccordion.scss

```diff
@@ -0,0 +1,58 @@
+@import "~vanilla-framework/scss/global_functions";
```

Comment:
Should probably get this file @lyubomir-popov 'd at some point. I imagine there's a lot of unnecessary styling that's now being handled in newer versions of Vanilla.